sql/moz-fx-data-shared-prod/mozilla_vpn_derived/fxa_attribution_v1/query.sql

```diff
@@ -29,10 +79,15 @@ WITH base AS (
         1
     )[SAFE_OFFSET(0)] AS attribution,
   FROM
-    firefox_accounts.fxa_content_auth_events
+    fxa_content_auth_stdout_events
   WHERE
-    IF(@date IS NULL, DATE(`timestamp`) < CURRENT_DATE, DATE(`timestamp`) = @date)
-    AND service = "guardian-vpn"
+    IF(@date IS NULL, partition_date < CURRENT_DATE, partition_date = @date)
+    -- cannot filter service because
+    AND (
+      service = "guardian-vpn"
+      -- service is missing for these event types
+      OR (service IS NULL AND (event_type = "fxa_rp_button - view" OR event_type LIKE "fxa_pay_%"))
```

Comment:
Will these event_types need to be updated in the future?

Reply:
probably, yeah